### DISCUSSION TITLE
css files missing

### DISCUSSION THREAD

**Discussion by yoscbd** [holger1411/understrap#1618] [Q&A]
I have installed and run the package using NPM.

i have try to override some of the defulte css via "_theme.scss"
I see that scss files are compiled and the relevant output css files are being generated and updated.

yet i dont see any of the changes and when i check the front end css sources i only see:

'https://127.0.0.1/guten2/wp-includes/css/dashicons.min.css

https://127.0.0.1/guten2/wp-includes/css/admin-bar.min.css?'

https://127.0.0.1/guten2/wp-includes/css/dist/block-library/style.min.css?ver=5.9.1

https://127.0.0.1/guten2/wp-content/themes/understrap/css/theme-bootstrap4.min.css?ver=1.1.0.1645789066

what can i do in order to reflect changes i take in the scss files to the front end?

**Comment by bacoords** [comment]
When you run `watch` the theme currently compiles the BS5 assets, but it doesn't compile the BS4 assets. See this thread for instructions: https://github.com/understrap/understrap/issues/1530#issuecomment-987388676

**Comment by yoscbd** [comment]
Thank you for a super quick reply,
I'll give it a try.
👍😀

**Reply by yoscbd** [reply]
it is working :) ...
Thank you.

Where in the scss folder should i add my own custom css changes if i want use bootstrap 5 with the "watch" command?

Also - if i use understrap to create my own custom theme, should i go for the child-them approach to build my new theme with my custom design or should i do everything via the parent theme as i am going to use my own design, functionality and theme template php files for posts and pages?

**Reply by bacoords** [reply]
the default `src/sass/theme/_theme.scss` is where you can add your CSS. That file is imported into the `src/sass/theme.scss` and `src/sass/theme-bootstrap4.scss` files.

We like to use child themes, then copy over any files from the parent theme that we plan on overriding. The main reason is so that some files we don't touch, like say WooCommerce templates, can receive new updates in the parent theme. However the child theme does include its own pitfalls.

**Reply by yoscbd** [reply]
"the default src/sass/theme/_theme.scss is where you can add your CSS. That file is imported into the src/sass/theme.scss and src/sass/theme-bootstrap4.scss files."

I think i didn't explain my self well... :)
I understand that the above instructions are  for bootstrap 4

- If I would like to use **bootstrap 5** where should I add my custom CSS code so it will be compiled?
when I edited "src/sass/theme/_theme.scss" and run the "watch" command I didn't see my changes being compiled.

**Reply by bacoords** [reply]
The same file. You can see the imports for the bootstrap 5 here: https://github.com/understrap/understrap/blob/main/src/sass/theme.scss

Then you'll want to make sure your site is loading theme.css and not theme-bootstrap4.css.
